webui/react/src/components/LogViewerTimestamp.tsx

```diff
@@ -239,18 +239,20 @@ const LogViewerTimestamp: React.FC<Props> = ({
   }, [ onDownloadClick ]);
 
   const handleEnableTailing = useCallback(() => {
+    if (direction === DIRECTIONS.TAILING) return;
```

Comment:
I think this would prevent "jump to bottom" behavior if you've already hit the arrow down button and you have scrolled up. 

or maybe more correctly tailing should stop when you scroll up. this used to be the case IIRC